src/store/user.js

```diff
@@ -23,7 +23,7 @@ const actions = {
       context.commit('SET_CAPABILITIES', { capabilities: null, version: null })
       // force redirect to login page after logout
       if (autoRedirect) {
-        router.push({ name: 'access-denied' })
+        router.push('/access-denied')
```

Comment:
from what I see the route name is "[accessDenied](https://github.com/owncloud/phoenix/blob/master/src/router/index.js#L52)", let's use that instead of the path ?

Reply:
Ah, you're right. I overlooked that 🤦‍♂ Thanks! Route name is back now